examples/06-dc/TwoCylinders_DipoleDipole_2Dinversion.py

```diff
@@ -0,0 +1,168 @@
+"""
+2D DC inversion of Dipole Dipole array
+======================================
+
+This is an example for 2D DC Inversion. The model consists of 2 cylinders,
+one conductive, the other one resistive compared to the background.
+
+We restrain the inversion to the Core Mesh through the use an Active Cells
+mapping that we combine with an exponetial mapping to invert
+in log conductivity space. Here mapping,  :math:`\\mathcal{M}`,
+indicates transformation of our model to a different space:
+
+.. math::
+    \\sigma = \\mathcal{M}(\\mathbf{m})
+
+Following example will show you how user can implement a 2D DC inversion.
+"""
+
+from SimPEG import (
+    Mesh,  Problem,  Survey,  Maps,  Utils,
+    EM,  DataMisfit,  Regularization,  Optimization,
+    InvProblem,  Directives,  Inversion
+    )
```

Comment:
would you mind closing this bracket at the beginning of the line? (eg. no indentation?)